src/tracker/psmove_tracker.c

```diff
@@ -31,7 +31,9 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <time.h>
+#define _USE_MATH_DEFINES
 #include <math.h>
+#undef _USE_MATH_DEFINES
```

Comment:
The `#undef` is probably not needed and could actually hurt (e.g. if some other include file later also does something depending on whether or not it is set).